src/varname.jl

```diff
@@ -148,6 +149,13 @@ function Accessors.set(obj, vn::VarName{sym}, value) where {sym}
     return Accessors.set(obj, PropertyLens{sym}() ⨟ getoptic(vn), value)
 end
 
+# Allow compositions with optic.
+function Base.:∘(optic::ALLOWED_OPTICS, vn::VarName{sym,<:ALLOWED_OPTICS}) where {sym}
+    return VarName{sym}(optic ∘ getoptic(vn))
+end
+function Accessors.opcompose(vn::VarName{sym,<:ALLOWED_OPTICS}, optic::ALLOWED_OPTICS) where {sym}
```

Comment:
This can be removed AFAICT, the fallback in https://github.com/JuliaFunctional/CompositionsBase.jl/blob/9c80024d40108ca873bf6351a14246c0165665d9/src/CompositionsBase.jl#L21 should be sufficient.
```suggestion
```